src/pages/Dashboard/Dashboard.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Container from '@mui/material/Container';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import { useDispatch } from 'react-redux';
+import { Logout } from '../../features/userSlice';
+
+function Dashboard() {
+  const dispatch = useDispatch();
+
+  const handleLogout = (e) => {
+    e.preventDefault();
```

Comment:
Is there any reason to `preventDefault`? Wouldn't setting `<button type="button" ...>` solve the problem?